### ISSUE TITLE
ml null

### ISSUE THREAD

**Issue by dhgomezg6** [adblockradio/adblockradio#10]
Hi Alexandre!
Following the instructions for installation and run the demo, I have this result:

`status={
	"liveResult": {
		"ml": null,
		"hotlist": null,
		"class": "9-unsure",
		"softmax": [
			0,
			0,
			0,
			0
		],
		"audio": "[redacted]",
		"predictorStartTime": 1562903441509,
		"metadata": null,
		"streamInfo": {
			"url": "http://cadena100-streamers-mp3.flumotion.com/cope/cadena100.mp3",
			"bitrate": 16000,
			"favicon": "https://d1v6pqrrmmnutj.cloudfront.net/post_square_images/attachments/000/000/821/original/cadena100.png?2015",
			"homepage": "http://www.cadena100.es",
			"audioExt": "mp3"
		},
		"playTime": 1562903445851,
		"tBuffer": 0
	}
`

I have downloaded the model of the Radio and put those files in model folder, but no prediction is made, should I do something else?

Also, I have this deprecation error, this is normal?

`[2019-07-12T03:50:44.473Z] error pred-ml: 	Spain_Cadena 100 mlpredict child stderr data: WARNING: Logging before flag parsing goes to stderr.
W0711 22:50:44.473676 140107082598208 deprecation_wrapper.py:119] From /media/daniel/Datos/tf/adblockradio/predictor-ml/mlpredict.py:57: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.
`

Thanks

**Comment by dest4** [comment]
Hi!

Model download should be automatic, no action is required on your side.
Could you please share the script you have used to launch the analysis?

**Comment by dhgomezg6** [comment]
I used the same demo.js, just changed the country and radioName

`
const { log } = require("abr-log")("demo");
const { Analyser } = require("./post-processing.js");

const country = "Spain";
const name = "Cadena 100";

log.info("start analyser!");
`

**Comment by dest4** [comment]
When it works well the log looks like this: https://gist.github.com/dest4/49eb1b71e74f7febf6b98b481ad4edd3
- Please post a similar full log, so we can better understand what is happening. Your hotlist field is null as well, that is worrying. Is it always null or intermittently null?
- Please share more about your platform, and the pip versions of keras and tensorflow (`pip show keras | grep Version` and `pip show tensorflow | grep Version`)
- Try with the option `JSPredictorMl` set to `true` (see documentation for reference), to bypass Python and any side effect of the warning you see.

**Comment by dhgomezg6** [comment]
Yes:
-  the full log is https://gist.github.com/Dinl/b1ba58c4812232e2b4dd7bffe2a51318 
- Not sure how to check the hostlist
- Keras Version: 2.2.4 and Tensorflow Version 1.14, Ubuntu 18.04
- With JSPredictorMl set to true, I see some difference, but not sure if should work in this way: https://gist.github.com/Dinl/ba62ff78825544d1aef7dff4a970a2a4
